pages/manage/productions/create.vue

```diff
@@ -199,23 +232,40 @@
         }
       },
       async linkEventsToNewProduction() {
-        await this.$api.productions.createWithEvents({
+        const response = await this.$api.productions.createWithEvents({
           name: this.productionName,
           eventIds: this.suggestedEventIdsWithoutProduction,
         });
+        if (response.status) {
```

Comment:
Shouldn't we also reset the messages to an empty array if the response of a re-try was succesfull?
Otherwise the message will stay visible and the suggestions won't be refreshed.

Reply:
I think that happens on line 254 in this file

Reply:
Yes indeed, my bad!